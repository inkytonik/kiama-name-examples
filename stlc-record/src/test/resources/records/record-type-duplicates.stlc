fun (p: {a:num, a:num}) { p }
