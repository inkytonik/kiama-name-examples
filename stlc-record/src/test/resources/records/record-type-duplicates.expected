src/test/resources/records/record-type-duplicates.stlc:1:1:info: fun (p : { ...: unknown
fun (p: {a:num, a:num}) { p }
^
src/test/resources/records/record-type-duplicates.stlc:1:6:info: p def: argument unknown
fun (p: {a:num, a:num}) { p }
     ^
src/test/resources/records/record-type-duplicates.stlc:1:10:info: a def: field in type num
fun (p: {a:num, a:num}) { p }
         ^
src/test/resources/records/record-type-duplicates.stlc:1:17:info: a def: field in type num
fun (p: {a:num, a:num}) { p }
                ^
src/test/resources/records/record-type-duplicates.stlc:1:27:info: p: unknown
fun (p: {a:num, a:num}) { p }
                          ^
unknown
